schedule/security/mariadb_ssl.yaml

```diff
@@ -0,0 +1,20 @@
+name: mariadb_ssl
+description:    >
+    mariadb test for sle, Leap and Tumbleweed
+schedule:
+    - boot/boot_to_desktop
```

Comment:
If this case run on s390, you need to load test module `bootloader_zkvm`

Reply:
Thanks, I'll update it.